### PR TITLE
Update service descriptions

### PR DESCRIPTION
**Pull request by rquarry**
This pull request is in reference to issue #1547. Fairly simple update, although I'm not sure whether usage of "ABRT" or "Automated Bug Reporting Tool" is preferred for the description. Most .service files I looked at did not have acronnyms, so that route was chosen. 

Impressed with the docs on the project! Looking forward to learning more and contributing.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
```
Comment (mgrabovsky) [inline CHANGELOG.md:8]: Remove this empty line, please.

**Review by mgrabovsky — CHANGES_REQUESTED** [review]
Thank you for the contribution, looks good overall. Could you please keep the “ABRT” in the messages, though? It can be interpreted as an acronym, but it's also sort intended not to be, and we almost never mention the “full” name, so I would stick with the wordmark for now.

File: CHANGELOG.md (hunk 1 above)
Comment (rquarry) [reply CHANGELOG.md:8]: Thank you for the feedback-I'm not hitched to any specific format. Just to be clear by keeping the "ABRT" you mean changing the description to:

```Description=ABRT coredumpctl message creator```

or were you referring to simply tacking on "ABRT" to what I originally submitted? I think the latter would make the strings a bit longer than they should be.

Comment (mgrabovsky) [reply CHANGELOG.md:8]: Yes, the original, please.

Comment (rquarry) [reply CHANGELOG.md]: Just pushed a new commit with requested changes

**Review by mgrabovsky — APPROVED** [review]
Great, thanks for your contribution!

**Pull request merged**